source/sgp_mode/SGPSymbiont.h

```diff
@@ -85,13 +86,16 @@ class SGPSymbiont : public Symbiont {
   void SetHost(emp::Ptr<Organism> host) {
     Symbiont::SetHost(host);
     if (my_host) {
+      if (cpu.state.used_resources) { cpu.state.used_resources.Delete(); }
+      if (cpu.state.shared_available_dependencies) {
+        cpu.state.shared_available_dependencies.Delete(); }
```

Comment:
Are we certain that these are pointers that the Symbiont owns? Aren't some of these potentially since being used by the old host? Or should be cleaned up by the old host upon death?

Reply:
I think the plan is to not delete these right? Leave them to the host to delete?